### ISSUE TITLE
Error in Electra model

### ISSUE THREAD

**Issue by dhruvladia** [gohjiayi/suicidal-text-detection#1]
I am getting this error:

The model did not return a loss from the inputs, only the following keys: logits. For reference, the inputs it received are input_ids,token_type_ids,attention_mask.

How do i fix this?

**Comment by gohjiayi** [comment]
Are you training your own ELECTRA model or predicting with the pretrained model? Able to share the code snippet?

**Comment by gohjiayi** [comment]
Closing due to inactivity. Please refer to `infer.ipynb` if you wish to predict with the finetuned ELECTRA model.
